service/conceptAnnotation.go

```diff
@@ -0,0 +1,16 @@
+package service
+
+// ConceptAnnotations models the annotations as it will be written on the queue
+type ConceptAnnotations struct {
```

Comment:
Naming - not sure what  ConceptAnnotation means. I understand what it is but the name is confusing as we always annotate with concepts. I think it's an Annotation (the binding) or maybe a MappedAnnotation.

Reply:
Renamed to `MappedAnnotations`